js/main.ts

```diff
@@ -0,0 +1,49 @@
+import {getRandomArrayElement, getRandomInt} from './utils/randomizers.ts';
+import {User, PhotoComment, Photo} from './contracts/common.ts';
+import {messages, names, descriptions} from './data/template-values.json';
+
+export const getGeneratedMocks = (amount: number):Photo[] => {
+	enum Default {
+		MinId = 1,
+		MinPhotoAddress = 1,
+		MaxPhotoAddress = 25,
+		MinLikes = 15,
+		MaxLikes = 200,
+		MinComments = 0,
+		MaxComments = 30,
+		MinAvatarId = 1,
+		MaxAvatarId = 6
+	}
```

Comment:
Ну блин. Обычные енамы дико неприятная вещь. Я бы советовал не юзать
```suggestion
const enum Default {
	MinId = 1,
	MinPhotoAddress = 1,
	MaxPhotoAddress = 25,
	MinLikes = 15,
	MaxLikes = 200,
	MinComments = 0,
	MaxComments = 30,
	MinAvatarId = 1,
	MaxAvatarId = 6
}

export const getGeneratedMocks = (amount: number): Photo[] => {

```

Reply:
Хотел попробовать с declare но забыл ))